api/controllers/UserController.js

```diff
@@ -228,10 +222,9 @@ module.exports = {
         promises.push(Ban.localBanUser(req.params.username));
       }
       Promise.all(promises).then(function () {
-        console.log('Process to ban /u/' + req.params.username + ' was completed successfully.');
+        sails.log('Process to ban /u/' + req.params.username + ' was completed successfully.');
```

Comment:
Yes, `sails.log()` is an alias for `sails.log.debug()`. ([Docs](http://sailsjs.org/documentation/concepts/logging/sails-log))